src/Aspire.Dashboard/Aspire.Dashboard.csproj

```diff
@@ -177,4 +177,8 @@
     <Compile Include="$(SharedDir)Model\KnownProperties.cs" Link="Utils\KnownProperties.cs" />
     <Compile Include="$(SharedDir)Model\KnownResourceTypes.cs" Link="Utils\KnownResourceTypes.cs" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <BlazorWebAssemblyLoadAllGlobalizationData>true</BlazorWebAssemblyLoadAllGlobalizationData>
```

Comment:
Does the dashboard use Blazor WebAssembly now?

Reply:
It doesn't. You're right that this isn't needed.

Reply:
@adamint did you test this PR? I tried the same code locally, minus the msbuild property here, and it didn't work for me.

Reply:
Yes, I tested this PR and it worked

Reply:
![image](https://github.com/dotnet/aspire/assets/20359921/fcaf3bad-d9cf-4dca-9bbf-65073338bc6e)
english browser, english OS

![image](https://github.com/dotnet/aspire/assets/20359921/10851592-70e3-4ae2-92cd-800d5d022200)
fr-FR browser, english OS

Reply:
>  I tried the same code locally, minus the msbuild property here, and it didn't work for me.

That's the reason it didn't work for you.

> It doesn't. You're right that this isn't needed.

This property _is_ needed. Otherwise, the functionality does not work. Note in the [documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/globalization-localization?view=aspnetcore-8.0#dynamically-set-the-culture-from-the-accept-language-header) that it should be set client-side, even if you are not writing blazor wasm

Reply:
We don't have a client-side app. We're not using WebAssembly. There is just a server app that is sending client updates via SignalR.

@SteveSandersonMS @danroth27 @javiercn @mkArtakMSFT Could you clarify what the msbuild property does and whether Aspire should need it.

Reply:
> >  I tried the same code locally, **minus the msbuild property here**, and it didn't work for me.
> 
> That's the reason it didn't work for you 😄 

Did you also verify that it doesn't work without this property? If so, then it looks like a bug in the docs or the product.

Reply:
Yes, I did verify that.

Reply:
@SteveSandersonMS @danroth27 @javiercn @mkArtakMSFT

Reply:
@drewnoakes we aren’t getting a response. Should I merge?

Reply:
I tested on my computer without the `BlazorWebAssemblyLoadAllGlobalizationData` property. The culture was correctly detected and applied to the UI.

This is what I launched Chrome with:

> C:\Program Files\Google\Chrome\Application> .\chrome.exe --accept-lang=fr-FR --user-data-dir=c:\Temp\Blah --lang=fr

Output:

![culture-wasmproperty](https://github.com/dotnet/aspire/assets/303201/842e769c-09c3-4bd4-8fa7-97736530ffd5)

Reply:
> @SteveSandersonMS @danroth27 @javiercn @mkArtakMSFT Could you clarify what the msbuild property does and whether Aspire should need it.

The `BlazorWebAssemblyLoadAllGlobalizationData` is only used when running on WebAssembly and configures the client-side runtime to load the entire globalization data set. This setting isn't needed when running from the server.